app/utilities.py

```diff
@@ -1,6 +1,28 @@
 from typing import Dict
+from app.data import MongoDB
 
+db = MongoDB("UnderdogDevs")
```

Comment:
An easy fix to removing this global variable is not obvious... you might want to consider refactoring your function to reduce the function's scope to only deal with bare inputs needed, perform the calculation, and then output an answer that the API can then serve up for consumption.